scripts/mea/et/internal_witness.py

```diff
@@ -307,15 +311,26 @@ def _parse_model_comments(self):
 
                     # Try match EMG comment
                     # Expect comment like /* TYPE Instance Text */
+
+                    match2 = new_emg_comment.search(text)
+                    if match2:
+                        data = json.loads(match2.group(1))
```

Comment:
Here you need to extract all options, including `comment`, `type`, etc. (see format `/* EMG_ACTION {"comment": "...", "name": "...", "relevant": true|false, "type": "..."} */`)